sources/packages/forms/src/form-definitions/studentprofile.json

```diff
@@ -3449,6 +3449,365 @@
       "addons": [],
       "lazyLoad": false
     },
+    {
+      "label": "HTML",
+      "labelWidth": "",
+      "labelMargin": "",
+      "tag": "h2",
+      "className": "category-header-large primary-color",
+      "attrs": [
+        {
+          "attr": "",
+          "value": ""
+        }
+      ],
+      "content": "Your consent",
+      "refreshOnChange": false,
+      "customClass": "",
+      "hidden": false,
+      "modalEdit": false,
+      "key": "html7",
+      "tags": [],
+      "properties": {},
+      "conditional": {
+        "show": false,
+        "when": "mode",
+        "eq": "student-edit",
+        "json": ""
+      },
+      "customConditional": "",
+      "logic": [],
+      "attributes": {},
+      "overlay": {
+        "style": "",
+        "page": "",
+        "left": "",
+        "top": "",
+        "width": "",
+        "height": ""
+      },
+      "type": "htmlelement",
+      "input": false,
+      "tableView": false,
+      "placeholder": "",
+      "prefix": "",
+      "suffix": "",
+      "multiple": false,
+      "defaultValue": null,
+      "protected": false,
+      "unique": false,
+      "persistent": false,
+      "clearOnHide": true,
+      "refreshOn": "",
+      "redrawOn": "",
+      "dataGridLabel": false,
+      "labelPosition": "top",
+      "description": "",
+      "errorLabel": "",
+      "tooltip": "",
+      "hideLabel": false,
+      "tabindex": "",
+      "disabled": false,
+      "autofocus": false,
+      "dbIndex": false,
+      "customDefaultValue": "",
+      "calculateValue": "",
+      "calculateServer": false,
+      "widget": null,
+      "validateOn": "change",
+      "validate": {
+        "required": false,
+        "custom": "",
+        "customPrivate": false,
+        "strictDateValidation": false,
+        "multiple": false,
+        "unique": false
+      },
+      "allowCalculateOverride": false,
+      "encrypted": false,
+      "showCharCount": false,
+      "showWordCount": false,
+      "allowMultipleMasks": false,
+      "addons": [],
+      "id": "exh9j0h"
+    },
+    {
+      "title": "Agreement (hides for student-edit)",
```

Comment:
(hides for student-edit) ??

Reply:
I will update the title. Although it was there before I started working on it so I did not see it.

Reply:
Updated